src/components/Files/FileUploadDialog.tsx

```diff
@@ -56,114 +69,78 @@ export default function FileUploadDialog({
           </DialogTitle>
         </DialogHeader>
         <div className="space-y-6 pr-5 max-h-[70vh] overflow-y-auto">
-          {isPdf ? (
```

Comment:
You have removed isPdf's checkbox functionality now 😅  always test/compare against what's in staging.